### DISCUSSION TITLE
【委托】被星际和平公司忽悠跑去买了千织屋的股票，结果刚好遇到最近第三次帝皇战争开打

### DISCUSSION THREAD

**Discussion by yakult-green-tea** [share121/inter-knot#293] [其他]
【委托】前阵子被星际和平公司忽悠跑去买了千织屋的股票，结果刚好遇到最近第三次帝皇战争开打，被套牢在高点了，现在口袋里的丁尼也剩没多少，连乔普师傅的拉面都吃不起了，有没有绳匠能推荐个悲剧题材的录像片，让我知道我还不是最凄惨的那个，或者来钱快的活也行

**Comment by AnbulesPeo** [comment]
六分街有一个叫RandomPlay的店，你可以问问他们店长的推荐，真的很靠谱！

**Comment by xunlie-xz** [comment]
建议去看《斯科特的一生》
